diff --git a/meson.build b/meson.build
index 46ad437..a805304 100644
--- a/meson.build
+++ b/meson.build
@@ -68,7 +68,6 @@ endif
 
 add_project_arguments(compiler_flags, language: 'c')
 
-dvdread_dep = dependency('dvdread', version: '>= 4.2.0')
 gio_unix_dep = dependency('gio-unix-2.0', version: '>= 2.31.0')
 gmodule_dep = dependency('gmodule-2.0')
 gtk_dep = dependency('gtk+-3.0', version: '>= 3.16.0')
diff --git a/src/disks/gducreatediskimagedialog.c b/src/disks/gducreatediskimagedialog.c
index 1b856e0..2a7a5f3 100644
--- a/src/disks/gducreatediskimagedialog.c
+++ b/src/disks/gducreatediskimagedialog.c
@@ -547,7 +547,7 @@ copy_span (int              fd,
 
   if (dvd_support != NULL)
     {
-      num_bytes_read = gdu_dvd_support_read (dvd_support, fd, buffer, offset, size);
+
     }
   else
     {
@@ -673,7 +673,7 @@ copy_thread_func (gpointer user_data)
           g_mutex_unlock (&data->copy_lock);
           g_idle_add (on_update_job, dialog_data_ref (data));
 
-          dvd_support = gdu_dvd_support_new (device_file, udisks_block_get_size (data->block));
+
 
           g_mutex_lock (&data->copy_lock);
           data->retrieving_dvd_keys = FALSE;
@@ -836,8 +836,6 @@ copy_thread_func (gpointer user_data)
     }
 
  out:
-  if (dvd_support != NULL)
-    gdu_dvd_support_free (dvd_support);
 
   data->end_time_usec = g_get_real_time ();
 
diff --git a/src/disks/meson.build b/src/disks/meson.build
index aa308dd..9e8ad99 100644
--- a/src/disks/meson.build
+++ b/src/disks/meson.build
@@ -17,7 +17,6 @@ sources = files(
   'gducrypttabdialog.c',
   'gdudevicetreemodel.c',
   'gdudisksettingsdialog.c',
-  'gdudvdsupport.c',
   'gduestimator.c',
   'gdufilesystemdialog.c',
   'gduformatdiskdialog.c',
@@ -82,7 +81,6 @@ sources += gnome.mkenums(
 )
 
 deps = [
-  dvdread_dep,
   gio_unix_dep,
   gmodule_dep,
   libcanberra_dep,
